ooni/nettests/experimental/tls_handshake.py

```diff
@@ -126,9 +129,11 @@ class HandshakeOptions(usage.Options):
          'Remote host IP address (v4/v6) and port, i.e. "1.2.3.4:443"'],
         ['port', 'p', None,
          'Use this port for all hosts, regardless of port specified in file'],
+        ['sni', None, None, 'Use SNI for Server Name Indication field'],
         ['ciphersuite', 'c', None ,
          'File containing ciphersuite list, one per line'],]
     optFlags = [
+        ['no-sni', None, 'Disable SNI'],
```

Comment:
A couple of things:

* I think we want to make it robust to running in unsupported versions. When it doesn't do SNI due to an out of date version we could log this in the report so that we can take this into account when we assess the results.

* The value of SNI, if it's not explicitly specified, should be inferred from the value of `--host` or from the `input`.